rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -385,10 +385,16 @@ def add_cookie(cookie)
         end
 
         def delete_cookie(name)
+          if ( name.nil? || name.empty?) then
+            raise Error::ArgumentError, 'Cookie name cannot be null or empty'
+          end
```

Comment:
Could you use a guard close here such as:

raise Error::ArgumentError, 'Cookie name cannot be null or empty' unless name && !name.empty?